checker/checkInductive.ml

```diff
@@ -165,7 +165,12 @@ let check_inductive env mind mb =
 
   check "mind_params_ctxt" (Context.Rel.equal Constr.equal mb.mind_params_ctxt mind_params_ctxt);
   ignore mind_universes; (* Indtypes did the necessary checking *)
-  ignore mind_variance; (* Indtypes did the necessary checking *)
+  check "mind_variance" (Option.equal (Array.equal Univ.Variance.equal)
+                           mb.mind_variance mind_variance);
+  check "mind_sec_variance" (match mind_sec_variance with
+      | None -> Option.is_empty mind_variance
```

Comment:
Can't we ensure that all on-disk inductive declarations have an empty  `mind_sec_variance` field?

Reply:
sure